hibike/hibike_message.py

```diff
@@ -104,6 +106,14 @@ def __str__(self):
     def __repr__(self):
         return str(self)
 
+    # pylint: disable=protected-access
+    def __eq__(self, other):
+        if not isinstance(other, HibikeMessage):
+            return False
+        return self._message_id == other._message_id and\
+               self._length == other._length and\
```

Comment:
does a `HibikeMessage` have a is_valid check or similar? If so, no need to compare the length with the payload